src/app/shared/player/player.component.html

```diff
@@ -12,8 +12,9 @@
     <section>
       <header>
         <div class="titles">
-          <h1 [title]="title">{{title}}</h1>
-          <h2 [title]="subtitle">{{subtitle}}</h2>
+          <h1 [title]="title"><a [href]="episodeLink" target="_top">{{title}}</a></h1>
```

Comment:
I might vote for just using plain non-links if `!episodeLink`, for now.  Then we can decide later if/what we should link to for plain feeds.